microsoft/testsuites/network/networksettings.py

```diff
@@ -485,7 +485,12 @@ def verify_device_rx_hash_level_change(self, node: Node, log: Logger) -> None:
             3. Validate changing the message level flag setting.
             4. Revert back the setting to original value.
         """,
-        priority=2,
+        priority=2,        
+        # BSD does not support the feature listed here and
+        # lacks the module checked in the support fucntion.
```

Comment:
1. Move it to above description, so it can show with skipped message. 
2. Specify which module is not in BSD.